packages/blade/package.json

```diff
@@ -21,7 +21,9 @@
     "tokens.d.ts",
     "tokens.js",
     "utils.d.ts",
-    "utils.js"
+    "utils.js",
+    "fonts",
```

Comment:
shouldn't fonts directory be part of build directory and we shall copy it inside rollup? the exports field you have mentioned doesn't work on RN yet and few other older versions of webpack

Reply:
It doesn't need to work on React Native right? anyway font installation steps will differ for react native.

Do we want to keep it inside build? I kept it outside so it doesn't need to be copied and be part of build as such.

Seems to work on POC - https://stackblitz.com/~/github.com/saurabhdaware/blade-self-hosted-font/tree/blade-import-css

Reply:
for now its fine but I think to make things predictable its better to keep it inside build. imagine like we do assets under `build/assets` so if tomorrow you take this build output for anything you don't have to keep reminding that "oh, fonts also needs to be copied separately"

Reply:
yup makes sense. Can do this later once our other bundling changes are done.

Reply:
can you note it down somewhere maybe some umbrella issue for all the todos?

Reply:
Yup added here - https://github.com/razorpay/blade/issues/1852